randovania/exporter/hints/joke_hints.py

```diff
@@ -29,4 +29,5 @@
     "We now pledge sole allegiance to our liberator, the great one, Dark Samus.",
     "Defeat enemies with the Light Beam to earn Dark Ammo. Defeat enemies with the Dark Beam to earn Light Ammo.",
     "Your Friend Roster is currently empty.",
+    "Raven Beak? I dont know him, but he sounds handsome, wouldn't you agree Samus?",
```

Comment:
`dont` -> `don't` ? Or are you really allowed to write it like that in english? Never seen it, honestly.

Reply:
Woops, forgot that my mobile sometimes forgets to do that. I'll fix that now.